index.html

```diff
@@ -804,12 +802,12 @@ <h4>Proof Verification (eddsa-rdfc-2022)</h4>
 
           <ol class="algorithm">
             <li>
-Let `publicKeyBytes` be the result of retrieving the
+Let |publicKeyBytes| be the result of retrieving the
 public key bytes associated with the
-`options`.`verificationMethod` value as described in the
+|options|.|verificationMethod| value as described in the
 [[[controller-document]]] specification,
 <a data-cite="controller-document#retrieve-verification-method">
```

Comment:
I think this phrasing (including the line deletion that follows) is better than what it replaces. And isn't the `controller-document` spec now the `controlled-identifier-document` (or `cid`) spec?
```suggestion
|options|.|verificationMethod| value as described in the
<a data-cite="controller-document#retrieve-verification-method">Retrieve
Verification Method</a> section of the [[[controller-document]]] specification.
```